core/src/components/range/range.tsx

```diff
@@ -922,6 +1014,7 @@ const renderKnob = (
           ev.stopPropagation();
         }
       }}
+      // onPointerMove={handlePointerMove}
```

Comment:
Was this left in intentionally?

Reply:
Great catch! https://github.com/ionic-team/ionic-framework/pull/29005/commits/3f4e38270582b8417d16e54a661bc93adebedc96